rebasehelper/build_tools/koji_tool.py

```diff
@@ -102,10 +109,28 @@ def _scratch_build(cls, source, **kwargs):
             logger.info('RPM build failed %s', weburl)
             logs.append(weburl)
             cls.logs.append(weburl)
-            raise BinaryPackageBuildError
+            return_code = cls.get_mock_return_code(build_error)
+            if return_code:
+                raise BinaryPackageBuildError(return_code=return_code)
+            else:
+                # Unknown error message obtained from koji
+                raise BinaryPackageBuildError()
         logs.append(weburl)
         return rpms, logs, task_id
 
+    @classmethod
+    def get_mock_return_code(cls, msg):
+        """
+        Parses mock status message for mock build return code
```

Comment:
Shouldn't this be rather `Parses koji status message for mock build return code`?